migration/iD12/postgresql/202410271703_IDEMPIERE-6278.sql

```diff
@@ -0,0 +1,225 @@
+
+UPDATE AD_Tab SET OrderByClause='AD_Column_Trl.AD_Language' WHERE AD_Tab_ID=681 /* Table and Column > Column Translation */;
+
+UPDATE AD_Tab SET OrderByClause='AD_Table_Trl.AD_Language' WHERE AD_Tab_ID=673 /* Table and Column > Table Translation */;
+
+UPDATE AD_Tab SET OrderByClause='AD_Task_Trl.AD_Language' WHERE AD_Tab_ID=126 /* Task > Task Translation */;
+
+UPDATE AD_Tab SET OrderByClause='C_TaxCategory_Trl.AD_Language' WHERE AD_Tab_ID=284 /* Tax Category > Translation */;
+
+UPDATE AD_Tab SET OrderByClause='C_Tax_Trl.AD_Language' WHERE AD_Tab_ID=458 /* Tax Rate > Translation */;
+
+UPDATE AD_Tab SET OrderByClause='C_Tax_Trl.AD_Language' WHERE AD_Tab_ID=53081 /* Tax Rate Parent > Translation */;
+
+UPDATE AD_Tab SET OrderByClause='TestUU_Trl.AD_Language' WHERE AD_Tab_ID=200371 /* Test UU Based Table > Test UU Based Table Trl */;
+
+UPDATE AD_Tab SET OrderByClause='C_UOM_Trl.AD_Language' WHERE AD_Tab_ID=273 /* Unit of Measure > Translation */;
+
+UPDATE AD_Tab SET OrderByClause='CM_Container_Trl.AD_Language' WHERE AD_Tab_ID=813 /* Web Project > Container Translation */;
+
+UPDATE AD_Tab SET OrderByClause='CM_Container_Element_Trl.AD_Language' WHERE AD_Tab_ID=815 /* Web Project > Container Element Translation */;
+
+UPDATE AD_Tab SET OrderByClause='CM_CStage_Trl.AD_Language' WHERE AD_Tab_ID=811 /* Web Project > Stage Translation */;
+
+UPDATE AD_Tab SET OrderByClause='CM_CStage_Element_Trl.AD_Language' WHERE AD_Tab_ID=817 /* Web Project > Stage Element Translation */;
+
+UPDATE AD_Tab SET OrderByClause='W_Store_Trl.AD_Language' WHERE AD_Tab_ID=711 /* Web Store > Store Translation */;
+
+UPDATE AD_Tab SET OrderByClause='W_MailMsg_Trl.AD_Language' WHERE AD_Tab_ID=713 /* Web Store > Message Translation */;
+
+UPDATE AD_Tab SET OrderByClause='AD_Field_Trl.AD_Language' WHERE AD_Tab_ID=115 /* Window, Tab and Field > Field Translation */;
+
+UPDATE AD_Tab SET OrderByClause='AD_Tab_Trl.AD_Language' WHERE AD_Tab_ID=116 /* Window, Tab and Field > Tab Translation */;
+
+UPDATE AD_Tab SET OrderByClause='AD_Window_Trl.AD_Language' WHERE AD_Tab_ID=117 /* Window, Tab and Field > Window Translation */;
+
+UPDATE AD_Tab SET OrderByClause='AD_Workbench_Trl.AD_Language' WHERE AD_Tab_ID=399 /* Workbench > Translation */;
+
+UPDATE AD_Tab SET OrderByClause='AD_WF_Node_Trl.AD_Language' WHERE AD_Tab_ID=123 /* Workflow > Node Translation */;
+
+UPDATE AD_Tab SET OrderByClause='AD_Workflow_Trl.AD_Language' WHERE AD_Tab_ID=149 /* Workflow > Workflow Translation */;
+
+UPDATE AD_Tab SET OrderByClause='C_Region_Trl.AD_Language' WHERE AD_Tab_ID=200128 /* Country Region and City > Translation */;
+
+UPDATE AD_Tab SET OrderByClause='M_PriceList_Trl.AD_Language' WHERE AD_Tab_ID=200130 /* Price List > Translation */;
+
+UPDATE AD_Tab SET OrderByClause='AD_PrintColor_Trl.AD_Language' WHERE AD_Tab_ID=200125 /* Print Color > Translation */;
+
```

Comment:
I think an issue with this type of dictionary update is there's no easy way for user to find out what has been updated by the migration scripts (there's possibility that it might overwrite some local changes). If we can't generate change log for this, perhaps at the minimum we should update the updatedby and updated date (Perhaps we can add a new 'Migration Script' user for this too).

Reply:
> I think an issue with this type of dictionary update is there's no easy way for user to find out what has been updated by the migration scripts (there's possibility that it might overwrite some local changes). If we can't generate change log for this, perhaps at the minimum we should update the updatedby and updated date (For updatedby, perhaps we can add a new 'Migration Script' user for this too).

Thanks @hengsin - added the Updated and UpdatedBy to the updates.

What you mention is a common problem any time applying a migration script, in order to discover that I developed the [Verify Migration process and window](https://wiki.idempiere.org/en/NF10_After_Migrate_Helper).  I've been using that process successfully and it shows when a migration script overwrote something customized.